OpenRA.Mods.Cnc/Traits/Render/WithBuildingBib.cs

```diff
@@ -35,8 +35,7 @@ public IEnumerable<IActorPreview> RenderPreviewSprites(ActorPreviewInitializer i
 			if (init.Contains<HideBibPreviewInit>(this))
 				yield break;
 
-			if (Palette != null)
-				p = init.WorldRenderer.Palette(Palette);
+			p = init.WorldRenderer.Palette(Palette);
```

Comment:
Can't `Palette` still be null (in case of null or an empty string)? 

Same below.

Reply:
It can, but `WorldRenderer` can handle `null` or empty strings properly now, so it's just going to return `null`. The goal was to simplify clients of `WorldRenderer.Palette()` method.